web/README.md

```diff
@@ -31,3 +31,9 @@ For example, to migrate to schema version 0.5.1, run:
 ```bash
 yarn migrate 0.5.1
 ```
+
+
+### Environment Variables
+
+- VUE_APP_SERVER_DOWNTIME_MESSAGE
+  - A custom error message to be displayed when the backend server can't be reached
```

Comment:
I don't much see the benefit in providing this as an env var. Is there a specific need for this? I'd like to avoid adding extra environment variables unless they're necessary.

Reply:
Yes, one of the requested features was to have a way to configure a custom message. During that Heroku outage a few weeks back, we wanted to display a message mentioning the specifics but had no way to quickly do that. Using an environment variable means we can go into the netlify gui/cli and just set it as opposed to needing to make a PR and cut a release.